### PR TITLE
Refactoring payments landing page

### PR DESCRIPTION
**Pull request by nnorbert**
Payments landing pageRefactoring

### CODE REVIEW

**Review by alfonsograziano — APPROVED** [review]
LGTM

File: apps/payments/app/[locale]/info/page.tsx [hunk 1]
```diff
@@ -9,6 +10,11 @@ import integration from "../../../public/landingPage/integration.png";
 import interfaceCitizen from "../../../public/landingPage/interfaceCitizen.png";
 import interfacePublicServant from "../../../public/landingPage/interfacePublicServant.png";
 import security from "../../../public/landingPage/security.png";
+import type { Metadata } from "next";
+
+export const metadata: Metadata = {
+  title: "Building Blocks",
```
Comment (alfonsograziano) [inline apps/payments/app/[locale]/info/page.tsx:16]: non-blocking: here we should have "Payments" instead of "Building Blocks"

Comment (nnorbert) [reply apps/payments/app/[locale]/info/page.tsx:16]: Ouch :| Sorry :D

Comment (nnorbert) [reply apps/payments/app/[locale]/info/page.tsx]: Done

**Pull request merged**